gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.0.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.8.3-all.zip
 zipStoreBase=GRADLE_USER_HOME
```

Comment:
Any reason not to use the latest version (7.0.2)?

Reply:
No reason other than I was being conservative since I was sure of your comfort level with version changes 😉 Ultimately I want to upgrade other dependencies all the way up to latest Kotlin 1.5 to see if the kotlinx serialization really is 2x faster. LMK if you're OK with it all in one go, or prefer piecemeal. Thanks!

Reply:
Let's do them all in one go 👍